docs/contributing/adding-images-to-the-wesite.md

```diff
@@ -0,0 +1,32 @@
+# Adding images to the design system website
+When adding or amending guidance on the design system website and you want to add an image, it's important to be considerate of the impact images have on the website. They can have implications for web performance and accessibility if used improperly.
```

Comment:
```suggestion
When deciding whether to add or amend guidance on the design system website with an image, it's important to consider the impact of using images. They can have implications for web performance and accessibility if used improperly.
```

Reply:
This edit changes the meaning of the content IMO. These docs are intended to help the team think about the impact of adding an image to guidance when writing guidance, not just editing content with images on. Negative performance impact normally occur at the point an image is added to a page, which this would help temper. I've add all the suggestions except the `with` and have tried to clean up the content.